components/eam/bld/namelist_files/use_cases/1950_eam_CMIP6.xml

```diff
@@ -23,91 +23,6 @@
 <prescribed_volcaero_type    >CYCLICAL						       </prescribed_volcaero_type    >
 <prescribed_volcaero_cycle_yr>1                                                        </prescribed_volcaero_cycle_yr>
 
```

Comment:
Is the large block below removed because those are now defaults?

Reply:
Exactly, we move all physics parameters to the default file.